### PR TITLE
PTX refactor, fixes and tests

### PR DESCRIPTION
**Pull request by FrederikBolding**
All around fixes to PTX + some refactoring.
- Undo functionality etc.
- Fixed issues with existing E2E tests
- Implemented small E2E test for send (PTX tests should be included when we have support for feature flags)

### CODE REVIEW

File: src/features/SendAssets/SendAssets.tsx [hunk 1]
```diff
@@ -39,7 +41,7 @@ function SendAssets() {
       component: IS_ACTIVE_FEATURE.PROTECT_TX ? SendAssetsFormWithProtectTx : SendAssetsForm,
       props: (({ txConfig }) => ({ txConfig }))(txFactoryState),
       actions: (payload: IFormikFields, cb: any) => {
-        if (getProTxValue(['state', 'protectTxEnabled'])) {
+        if (getProTxValue(['state', 'protectTxEnabled']) && !isMyCryptoMember) {
```
Comment (vilsbole) [inline src/features/SendAssets/SendAssets.tsx:44]: How about we change this syntax and access the values directly from the context?
```
const { PTxEnabled } = useContext(PTxContext)
```

File: src/features/ProtectTransaction/ProtectTxProvider.tsx [hunk 2]
```diff
@@ -8,10 +8,11 @@ import {
   GetTokenTxResponse
 } from '@services/ApiService';
 import { AssetContext, getAssetByUUID, StoreContext } from '@services/Store';
+import { NansenService, NansenServiceEntry } from '@services/ApiService/Nansen';
 import { useScreenSize } from '@utils';
+import { WALLETS_CONFIG } from '@config';
 
-import { WALLETS_CONFIG } from '../../config';
-import { NansenService, NansenServiceEntry } from '@services/ApiService/Nansen';
+import { IS_ACTIVE_FEATURE } from '../../config/isActiveFeature';
```
Comment (vilsbole) [inline src/features/ProtectTransaction/ProtectTxProvider.tsx:15]: Import from @config please

File: src/features/SendAssets/components/SendAssetsForm.tsx [hunk 3]
```diff
@@ -333,11 +338,7 @@ const SendAssetsForm = ({ txConfig, onComplete }: IStepComponentProps) => {
   const userAccountEthAsset = userAssets.find((a) => a.uuid === ETHUUID);
 
   return (
-    <div
-      className={`SendAssetsForm ${
-        getProTxValue(['state', 'mainComponentDisabled']) ? 'SendAssetsForm-disabled' : ''
```
Comment (vilsbole) [inline src/features/SendAssets/components/SendAssetsForm.tsx:338]: Did you check if we can remove these class definitions as well?

File: src/features/ProtectTransaction/ProtectTxProvider.tsx (hunk 2 above)
Comment (FrederikBolding) [reply src/features/ProtectTransaction/ProtectTxProvider.tsx:15]: It's imported directly to prevent circular dependency issues.

**Pull request merged**